web/src/App.jsx

```
@@ -56,6 +56,41 @@ const ContactIcon = () => <svg width="24" height="24" viewBox="0 0 24 24" fill="
+      return () => clearTimeout(timer);
+    }
+  }, [copied]);
+
+  return (
+    <button
+      className="ghost-btn icon-only"
+      onClick={handleCopy}
+      aria-label={label || "Copy to clipboard"}
+      title={label || "Copy"}
+      style={{ padding: 4 }}
+    >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)